kuma/payments/forms.py

```diff
@@ -137,3 +197,137 @@ def make_charge(self):
                     )
                 )
         return False
+
+
+class RecurringPaymentForm(ContributionForm):
+
+    def __init__(self, *args, **kwargs):
+        super(RecurringPaymentForm, self).__init__(*args, **kwargs)
+        self.fields['donation_choices'].choices = RECURRING_PAYMENT__CHOICES
+
+    @staticmethod
+    def create_customer(email, token, user, name):
+        customer = stripe.Customer.create(
```

Comment:
Maybe the name should be used as the Customer description?

```
customer = stripe.Customer.create(
    description=name,
    email=email,
    ...
)
```

Reply:
Actually it is a good idea 👍